kingpin/actors/base.py

```diff
@@ -511,7 +511,8 @@ def _generate_escaped_url(self, url, args):
     # garbled data (ie, maybe a 500 errror or something else thats not in
     # JSON format, we should back off and try again.
     @gen.coroutine
-    def _fetch(self, url, post=None, auth_username=None, auth_password=None):
+    def _fetch(self, url, post=None, auth_username=None, auth_password=None,
+               method='GET'):
```

Comment:
I could undo this since I switched to the rest client, but I figure I might as well leave it in.